[rospy.client][INFO] 2017-11-10 12:08:13,212: init_node, name[/rostopic_4470_1510333693194], pid[4470]
[xmlrpc][INFO] 2017-11-10 12:08:13,212: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2017-11-10 12:08:13,213: Started XML-RPC server [http://mdas-Meerkat:34049/]
[rospy.init][INFO] 2017-11-10 12:08:13,213: ROS Slave URI: [http://mdas-Meerkat:34049/]
[rospy.impl.masterslave][INFO] 2017-11-10 12:08:13,213: _ready: http://mdas-Meerkat:34049/
[rospy.registration][INFO] 2017-11-10 12:08:13,213: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2017-11-10 12:08:13,213: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2017-11-10 12:08:13,313: registered with master
[rospy.rosout][INFO] 2017-11-10 12:08:13,313: initializing /rosout core topic
[rospy.rosout][INFO] 2017-11-10 12:08:13,317: connected to core topic /rosout
[rospy.simtime][INFO] 2017-11-10 12:08:13,319: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2017-11-10 12:08:13,379: topic[/gps_fix_psr] adding connection to [http://mdas-Meerkat:33163/], count 0
[rospy.internal][INFO] 2017-11-10 12:08:13,436: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2017-11-10 12:08:20,801: signal_shutdown [signal-2]
[rospy.internal][INFO] 2017-11-10 12:08:20,807: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2017-11-10 12:08:20,808: topic[/gps_fix_psr] removing connection to http://mdas-Meerkat:33163/
[rospy.internal][WARNING] 2017-11-10 12:08:20,810: Unknown error initiating TCP/IP socket to mdas-Meerkat:36343 (http://mdas-Meerkat:33163/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2017-11-10 12:08:20,810: Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/topics.py", line 307, in close
    c.close()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 807, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2017-11-10 12:08:20,810: signal-2
[rospy.core][INFO] 2017-11-10 12:08:20,975: signal_shutdown [atexit]
